qbraid_qir/qasm3/profiles/adaptive_execution.json

```diff
@@ -0,0 +1,159 @@
+{
```

Comment:
Can you add a `template.json` for the structure which you have chosen? For example, allowed keys, types, values etc.